web/test/pages/onboarding/onboarding-starting.test.tsx

```diff
@@ -318,22 +276,7 @@ describe("onboarding - starting a business", () => {
     });
   });
 
-  it("removes required fields error when user goes back", async () => {
```

Comment:
This test isn't necessarily testing legal structure, it's testing banner behavior and using legal structure as an example. 

I think we need to keep this test but just use a different banner error (if there is one?)